modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlAsynchronousApiTest.java

```diff
@@ -345,14 +344,17 @@ public void checkTransactionsWithDml() {
         outerTx.commit();
 
         // Outdated tx.
-        Transaction outerTx0 = outerTx;
-        IgniteException e = assertThrows(IgniteException.class,
```

Comment:
The test check seems to be working, why do we need to disable it right now? Maybe just add a TODO to rework it?

Reply:
ok